src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -859,9 +958,53 @@ internal void HandleWorkerMetadataRequestError(Exception exc)
             _eventManager.Publish(new WorkerErrorEvent(_runtime, Id, exc));
         }
 
+        private ValueTask SendStreamingMessageAsync(StreamingMessage msg)
+        {
+            if (_outbound is null)
+            {
+                return default;
```

Comment:
For this scenario, particularly if the message was an invocation request, shouldn't we fail here? In the case of an invocation, will will the behavior be if we simply return? I'm assuming the invocation would remain in-flight until the function timeout (which could be quite large) occurs.

Reply:
With a change above, we shouldn't be able to get in this state at all - going to remove these extra checks since it's no longer nullable and checked in the constructor.